src/main/resources/js/creator/creator-utilities.js

```diff
@@ -204,6 +205,29 @@ function setCommentOnIssueUpdatesChoice(commentOnIssueUpdatesChoice) {
     }
 }
 
+function setWorkflowStatus(workflowStatusSpan, newStatus) {
+    console.log("workflowStatusElement: " + workflowStatusSpan);
+    while (workflowStatusSpan.firstChild) {
+        workflowStatusSpan.removeChild(workflowStatusSpan.firstChild);
+    }
+    workflowStatusSpan.appendChild(document.createTextNode(newStatus));
+
+    const parentDivClassList = workflowStatusSpan.parentElement.classList;
+    if (newStatus === 'Enabled') {
+        parentDivClassList.add('workflowStatusEnabled');
+        parentDivClassList.remove('workflowStatusPartial');
+        parentDivClassList.remove('workflowStatusDisabled');
+    } else if (newStatus.includes('only')) {
```

Comment:
Wouldn't it be better to look for Enabled and Disabled? Since we may add more middle states in the future, we could add more issue types

Reply:
Yes, the original logic also had to handle errors when this was global, so I wanted that case to end up in the else as well.